composer.json

```diff
@@ -7,6 +7,34 @@
         "coding standard"
     ],
     "require": {
-        "squizlabs/php_codesniffer": "^2.7"
+        "squizlabs/php_codesniffer": "3.0.1"
```

Comment:
Please note we point here specific version, without `^`/`~`. It guarantees that we always have the same sniffs rules with specific version of `zend-coding-standard`. Otherwise it would be possible that the same version of `zend-coding-standards` gives different results of phpcs.